### PR TITLE
Db.python.plan

### PR DESCRIPTION
**Pull request by davebambrick**
### Background

We want to provide an example client implementation to start and retrieve a motion plan.

### What's new

- [ ] [New feature description]

### Related work

- [ ] [link] needs this PR
- [x] this PR needs https://github.com/DexaiRobotics/planning_service/pull/117

### TODOs / Nice-To-Haves

- [ ] [Add system tests for new feature.]

### CODE REVIEW

File: examples/planning/python/planner_client.py [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+from build.basic_types_pb2 import PlanContextId, ProblemDef, SystemConf, Conf, SysPoly
+from build.planner_pb2 import (
+    StartPlanRequest,
+    StartPlanResponse,
+    RetrievePlanRequest,
+    RetrievePlanResponse,
+)
+from build.planner_pb2_grpc import MotionPlannerStub
+
+K = 4
```
Comment (davebambrick) [inline examples/planning/python/planner_client.py:25]: magic number - remove or document

File: examples/planning/python/planner_client.py [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+            raise ValueError(
+                f"Polynomial is undefined for t={t} outside [{self.start}, {self.end}]!"
+            )
+        t_vec = np.array([[1, t, t**2, t**3]])
+        if t == self.start:
+            P_idx = 0
+        elif t == self.end:
+            P_idx = self.P_arr.shape[0] - 1
+        else:
+            P_idx = np.searchsorted(self.breaks[:-1], t) - 1
```
Comment (davebambrick) [inline examples/planning/python/planner_client.py:66]: docs [here](https://numpy.org/doc/stable/reference/generated/numpy.searchsorted.html). Essentially a binary search in order to grab the interval on which the sample time `t` should be evaluated.

File: examples/planning/python/planner_client.py (hunk 1 above)
Comment (sadraddini) [reply examples/planning/python/planner_client.py:25]: Address this

**Review by sadraddini — APPROVED** [review]
Merge when ready

**Pull request merged**